NMEA2000.cpp

```diff
@@ -534,7 +537,13 @@ void tNMEA2000::HandleISORequest(const tN2kMsg &N2kMsg) {
       case 126998L: /* Configuration information */
         SendConfigurationInformation(iDev);
         break;
-       // ToDo: Write default handler so that user can add their own.
+      default:
+         if (RqstHandler!=0 ) RqstHandler(N2kMsg);     /* See if user wants to handle it */
```

Comment:
Maybe the request handler could return true/false to indicate whether they processed the message.

If the `RqstHandler` returns `true`, you do nothing more (like you are doing now). If it returns `false`, you could then send a standard ACK.